scylla/src/macros.rs

```diff
@@ -1,6 +1,7 @@
-/// #[derive(FromRow)] derives FromRow for struct
+/// Derive macro for the [`FromRow`](crate::frame::response::cql_to_rust::FromRow) trait
+/// which deserializes a row to given Rust structure.
 ///
-/// Works only on simple structs without generics etc
+/// It is supported for structs with either named or unnamed fields.
```

Comment:
Please keep mention of the limitation to structs without generics.

Reply:
> I removed this mention for a reason. IIRC, the following compiles:
> 
> ```rust
> #[derive(FromRow)]
> struct GenericRow<T: FromCqlVal<CqlValue>> {
>     col: T,
> }
> ```
> 
> Do you see any other use cases for generics in structs that would derive `FromRow`?

What about if you remove `FromCqlVal<CqlValue>` bound?
Typical derives (`Clone` etc) work in this case:
```rust
#[derive(Clone)]
struct Generic<T> {
   val: T,
}
```

Then `Clone` will be implemented only for `T`'s that implement `Clone`. Does `FromRow` work the same way? If not, then it doesn't really support generics.

Reply:
In such case, we obviously get the error that `T` does not implement `FromCqlVal<CqlValue>` since it's required to deserialize the value.

> Typical derives (Clone etc) work in this case

Now that I think about it, you are right. I'll keep the mention then.

Reply:
One way we could solve it is to generate the bounds for generics ourselves, but it's not related to this PR + I don't think it's so important to support generics anyway.